website/docs/r/glue_connection.html.markdown

```diff
@@ -71,6 +71,7 @@ The following arguments are supported:
 In addition to all arguments above, the following attributes are exported:
 
 * `id` - Catalog ID and name of the connection
+* `id` - The ARN of the Glue Connection.
```

Comment:
small typo: i think you meant to refer to `arn`?

Reply:
Done